modules/ossm-observability-access.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+//- ossm-observability.adoc
+
+[id="ossm-observability-access-console_{context}"]
+= Accessing the Kiali console
+
+To access the console, in the menu bar, click the *Application launcher* > *Kiali*.
+
+Select the project for your service in the *Namespace* field. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console. If you have installed the Bookinfo example, select `bookinfo`.
```

Comment:
Are these out of order?  You should be logging in before you're selecting a namespace.  Unless you're making that selection while you're still in OpenShift?

Reply:
made a change